src/Raven.Server/Web/RequestHandler.cs

```diff
@@ -250,6 +250,27 @@ protected bool IsWebsocketRequest()
             return HttpContext.WebSockets.IsWebSocketRequest;
         }
 
+        protected string GetRaftRequestIdFromQuery()
+        {
+            var guid = GetStringQueryString("raft-request-id", required: false);
+
+            if (RequestRouter.TryGetClientVersion(HttpContext, out var clientVersion))
+            {
+                if (clientVersion.Build <= 42_008) // TODO: what should be the behaviour with the nightly version 
```

Comment:
The nightly also have a version indication, no?

Reply:
IIRC nightlys are simply 42

Reply:
Should be `42_009` ?